tools/reactive-controllers/src/FocusGroup.ts

```diff
@@ -145,6 +155,24 @@ export class FocusGroupController<T extends HTMLElement>
         );
     }
 
+    changeDefaultItemFocus(): void {
+        this.clearElementCache();
+        let diff = 0;
+        if (this.currentIndex === this.elements.length) {
+            diff = -1;
+        }
```

Comment:
```suggestion
        const diff = this.currentIndex === this.elements.length ? -1 : 1;
```
Does this achieve the same?

Reply:
nope, it should be 
`const diff = this.currentIndex === this.elements.length ? -1 : 0;`

Reply:
done